experiments/compositions/samples/AttachedAKS/03-attached-1.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: facade.compositions.google.com/v1
+apiVersion: compositions.mycompany.com/v1
```

Comment:
we are using idp instead of compositions for the group.
Not a blocker, but just wanted to let you know.

Reply:
`idp.mycompany.com`

Reply:
Didn't realize. I've updated to match.